core/templates/dev/head/dashboard/NotificationsDashboard.js

```diff
@@ -24,6 +24,11 @@ oppia.controller('DashboardNotifications', [
       notificationType === 'feedback_thread' ? '#/feedback' : '');
   };
 
+  $scope.getItemUrl = function(activityId, notificationType) {
+    return '/create/' + activityId + (
+      notificationType === 'feedback_thread' ? '#/feedback' : '');
```

Comment:
I don't know why this is happening but when I tried this , it did not behave as a link.
```
return '/create/' + activityId + (
  notificationType === 'feedback_thread' ? '#/feedback' : '');
``` is working 

```
return (
  '/create/' + activityId + 
  notificationType === 'feedback_thread' ? '#/feedback' : '');
``` is not working